src/server/user-controller.js

```diff
@@ -228,6 +181,53 @@ class UserController {
     return res.send({ lockedOut: newLockedOutState });
   }
 
+  registerMiddleware(router) {
```

Comment:
You extracted the registered middleware in `i18n-controller.js` but not here. It's not a huge inconsistency that would bother me, I just wanted to mention it. Maybe there's also no real benefit in extracting the registration of middlewares into their own `handlers` unless we want to test it with mocked `req`s and `res`s and `next`s (which we don't do in this case here).

Reply:
Here I didn't want to overcomplicate it, because the handlers are basically already either `session` or the `passport` methods. Also did not want to risk introducing bugs here. However I wanted to have the middleware handler registration at least in the same spot as in the other controllers.

For `user-agent-controller` and `i18n-controller` I think we could use some middleware handlers tests.
`settings-controller` I just extracted for consistency with the others..

Reply:
I see, just wanted to mention it. I think it's fine.